cluster/gce/trusty/configure-helper.sh

```diff
@@ -182,6 +184,16 @@ assemble_kubelet_flags() {
   echo "KUBELET_OPTS=\"${KUBELET_CMD_FLAGS}\"" > /etc/default/kubelet
 }
 
+start_kubelet(){
```

Comment:
Ah, scratch the second question, it's called based on what's in the yaml configs below :)